test/src/099-http_tracing/main

```diff
@@ -0,0 +1,80 @@
+cvmfs_test_name="HTTP Tracing with Custom Headers"
+cvmfs_test_autofs_on_startup=false
+cvmfs_test_suites="quick"
+
+source ./src/099-http_tracing/setup_teardown
+
+TEST099_REPO=lhcb.cern.ch
+TEST099_MOUNTPOINT=/cvmfs/lhcb.cern.ch
+
+CVMFS_TEST_099_TMPFILE=
+
+mount_and_read() {
+  echo "Mount repo, read and check HTTP tracing headers"
+
+  # used in file test_functions for cvmfs_moount()
+  CVMFS_TEST_DEBUGLOG="/tmp/$CVMFS_TEST_099_TMPFILE"
+  CVMFS_TEST_QUOTED_PARAM_KEY_VAL='CVMFS_HTTP_TRACING_HEADERS="h1:test|CVMFS-X-h2:ff|X-CVMFS-h3:12_ad |  h4  : 12fs_? "'
+
+  echo "   Mount $TEST099_REPO"
+  cvmfs_mount $TEST099_REPO "CVMFS_HTTP_TRACING=on" || return 1
+
+  echo "   Client config $TEST099_REPO"
+  cat /etc/cvmfs/default.local
+
+
+
+  echo "   Access data to send HTTP requests"
+  cat $TEST099_MOUNTPOINT/*.sh > /dev/null
```

Comment:
This may break. You can use `find  $TEST099_MOUNTPOINT -type f | tail -n1` to get a file to print.